gatorgrouper/urls.py

```diff
@@ -1,9 +1,19 @@
 """ This is undocumented """
-# from django.urls import path,
-from django.urls import re_path
+from django.urls import path
+
+# from django.urls import re_path
 
 # from django.conf import settings
 # from django.conf.urls.static import static
 from . import views
 
-urlpatterns = [re_path(r"^$", views.index, name="index")]
+urlpatterns = [
+    path("index", views.index, name="index"),
+    path(
+        "", views.home, name="Gtorgrouper-home"
+    ),  # first attribute is a space, meaning homepage
+    path("classes", views.create_classes, name="Gtorgrouper-classes"),
```

Comment:
There are two misspellings of "Gatorgrouper" here.

Reply:
saijin don't leave! I am here I will do exactly what you tell me to do!